core/src/main/java/de/danoeh/antennapod/core/storage/FeedSearcher.java

```diff
@@ -25,19 +25,20 @@
      */
     public static List<SearchResult> performSearch(final Context context,
                                                    final String query, final long selectedFeed) {
-        final int values[] = {0, 0, 1, 2};
-        final String[] subtitles = {context.getString(R.string.found_in_shownotes_label),
-                context.getString(R.string.found_in_shownotes_label),
+        final int values[] = {2, 1, 0, 0};
```

Comment:
Yes,
The code used a for loop (int i = 0; i < 4; i++) to run the 4 search tasks. It was initially set up to run the shownotes search first and the title search last. I reordered it so that the title search ran first. This way, in case of duplicates, the title result would be matched first and subsequent matches would be excluded. I made this decision, since the sorting of the search results already showed preference to title results by listing them before shownotes results.

I was debating making the search result include all the matches. For example, if a user searches for "steel" and the word is used in the title and shownotes, the result have "Found in title, shownotes" instead of just "Found in title". Do you think this is a worthwhile feature or very beneficial?